cluster-autoscaler/cloudprovider/openshiftmachineapi/machineapi_controller.go

```diff
@@ -276,17 +277,16 @@ func (c *machineController) machinesInMachineSet(machineSet *MachineSet) ([]*Mac
 	return result, nil
 }
 
-// getAPIVersionGroup returns a string that specifies the version and
-// group for the API. It will return either the value from the
-// MACHINEAPI_VERSION_GROUP environment variable, or the default
-// value. This value will be a string in the format of "version.group".
-func getAPIVersionGroup() string {
-	vg := os.Getenv(machineAPIVersionGroupEnvVar)
-	if vg == "" {
-		vg = defaultMachineAPIVersionGroup
-	}
-	klog.V(4).Infof("Using API Version Group %v", vg)
-	return vg
+// getMachineAPIGroup returns a string that specifies the group for the API.
+// It will return either the value from the
+// MACHINEAPI_GROUP environment variable, or the default value.
+func getMachineAPIGroup() string {
+	g := os.Getenv(machineAPIGroupEnvVar)
+	if g == "" {
+		g = defaultMachineAPIGroup
+	}
+	klog.V(4).Infof("Using API Group %v", g)
```

Comment:
quick question, since we know this log statement (and similar below) take a string type in the format, should the `%v` be changed to `%q`?

Reply:
I can do that. I'd generally prefer to use %v all around unless there's a good reason to not. It's less churn in the fmt strings if you ever change the type.

Reply:
I think there are cases where `%q` makes more sense, and I think the group falls into that camp. Here I would rather see: `"foo"`.

Reply:
thanks for the explanations, part of the reason for my question is to help me understand what is more idiomatic in the go code we write.

Reply:
I tend to prefer using the actual typed format strings (eg `%s`, `%d` etc) but it's idiomatic to use `%v` for error types I believe, I'd normally do like a `fmt.Errorf("blah %s failed: %v", "string", err)` sort of thing if I'm constructing errors. YMMV